WordPress/Classes/ViewRelated/Post/Revisions/Browser/RevisionDiffsBrowserViewController.swift

```diff
@@ -1,25 +1,33 @@
 import Gridicons
 
-class RevisionBrowserState {
+class RevisionBrowserState: SelectedRevisionLoadedProtocol {
+    let post: AbstractPost?
```

Comment:
Seems that `post` is here just for access to `Blog`. Would be more direct to put the `blog` here.

However, I think that's necessary because `RevisionDiffsBrowserViewController` is doing the revision loading. If we did the load in `RevisionsTableViewController` we could ditch the `post` property entirely. I'd personally really prefer that, as it would simplify `RevisionDiffsBrowserViewController` and (imo) make `RevisionsTableViewController` a bit easier to understand as well.

Let me know if I'm completely off base here.